### PR TITLE
Feat/billing standalone app

### PR DESCRIPTION
**Pull request by frenautvh**
| Question         | Answer
| ---------------- | ---
| Branch?          | develop
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | internal MANAGER-5821
| License          | BSD 3-Clause

## Description

add billing standalone application

- [x] TRANS-36331

### CODE REVIEW

**Review by zakariasahmane — DISMISSED** [review]

**Review by Jisay — DISMISSED** [review]

**Comment by frenautvh** [comment]
rebased on develop 2e786868a1d4b53d5c4237c8f798ca9efd9c04a4

**Review by zakariasahmane — DISMISSED** [review]

**Review by Jisay — APPROVED** [review]

**Review by cbourgois — APPROVED** [review]

**Comment by marie-j** [comment]
@frenautvh I see translations added in fr_FR but no linked request, is it legit ?

File: packages/manager/modules/billing/src/billing.scss [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import 'bootstrap4/scss/_functions.scss';
```
Comment (marie-j) [inline packages/manager/modules/billing/src/billing.scss:1]: Not scoped to billing module ? we might encounter issues if imported in Dedicated with bootstrap3

**Comment by marie-j** [comment]
Hello @frenautvh ,
Could you please solve existing conflicts ? 
Thanks

**Review by JeremyDec — DISMISSED** [review]

**Comment by frenautvh** [comment]
rebase on develop 6c095523b9b74d1eff1abcc874a51cc5a79068c1

**Review by Jisay — DISMISSED** [review]

**Pull request closed without merging**